docs/user/visual-diff.rst

```diff
@@ -1,46 +1,96 @@
+
+.. figure:: /img/build-overview-comment.png
+
+General settings
+----------------
+
+Base version
+~~~~~~~~~~~~
+
+The base version is the version of the documentation that is used to compare against the pull request.
+By default, this is the latest version of the documentation.
+
+.. note::
+
+   This option can be changed by contacting :doc:`/support`.
 
-Visual diff will show all the sections that have changed and their differences highlighted with red/green background colors.
-Then you can jump between each of these chunks by clinking on the up/down arrows.
+Ignored files
+~~~~~~~~~~~~~
 
-Configuring Visual Diff
------------------------
+You can configure a list of files or patterns to be ignored when listing the files that changed in the pull request.
 
-All the available configuration for the visual diff addon can be found under :guilabel:`Settings > Addons > Visual diff` in the :term:`dashboard`.
-You can choose to disable/enable the `Visual diff` feature on a per project basis.
-Visual diff can also ignore files by specifying these files under `File tree diff`, regex supported.
+#. Go the :guilabel:`Settings` tab of your project.
+#. Click on :guilabel:`Addons`, and click on :guilabel:`File tree diff`.
+#. In the :guilabel:`Ignored files` field, add one or more patterns to ignore, one per line.
+#. Click on :guilabel:`Save`.
 
-Troubleshooting Visual diff
----------------------------
+Patterns are matched against the relative paths of the HTML files produced by the build,
+you should try to match ``index.html``, not ``docs/index.rst``, nor ``/en/latest/index.html``.
+Patterns can include one or more of the following special characters:
 
-Visual diff only works when we detect changes on the page,
-so ensure you are on a page that has changed in the current pull request.
+- ``*`` matches everything, including slashes.
+- ``?`` matches any single character.
+- ``[seq]`` matches any character in ``seq``.
 
-There are also some known issues that currently don't display properly.
-We are working to improve the UX, but so far we've found the following issues:
+Limitations and known issues
+----------------------------
 
-* **Tables** are shown to have changes when they may not have changed. This is due to subtle variations in how HTML tables are rendered, and will be fixed in a future version.
-* **Invisible changes** sometimes are marked as diff due than the underlying HTML changing, but there is no visual change. This could happen if the URL of a link changed, for example.
-* **Chunks background is incorrect** when we are unable to detect the correct main parent element for the chunk.
+- The diff considers HTML files only.
+- The diff is done between the files from the latest successful build of the pull request and the default base version (latest by default).
+  If your pull request gets out of sync with its base branch, the diff may not be accurate, and may show unrelated files and sections as changed.
```

Comment:
I'd really love to find a better way to handle this, as it's the #1 UX issue with it that I see regularly. Not sure how to really fix it tho, perhaps using git to see what files have changed between the base `main` 🤷

Reply:
I opened an issue about this some time ago https://github.com/readthedocs/readthedocs.org/issues/12232